tests/jax/latest/wmt.libsonnet

```diff
@@ -33,10 +38,17 @@ local tpus = import 'templates/tpus.libsonnet';
   },
   local wmt = common.runFlaxLatest {
     modelName:: 'wmt',
-    extraDeps+:: ['tensorflow_text sentencepiece'],
+    extraDeps+:: ['tensorflow_text sentencepiece', 'protobuf==3.20.*', 'tensorflow_datasets'],
+  },
+  local wmt_profiling = wmt {
+    local config = self,
+    testScript+:: |||
+      gsutil -q stat $(MODEL_DIR)/plugins/profile/*/*.xplane.pb
+    ||| % (self.scriptConfig {}),
   },
   configs: [
     wmt + functional + v2_8,
     wmt + convergence + v3_8 + timeouts.Hours(20),
+    wmt_profiling + profile + v3_8 + timeouts.Hours(1),
```

Comment:
How long has the profiler test been taking to run? Just wondering if the timeout could be cut to something like 20-30 minutes instead

Reply:
Takes about 15-20 mins as per my observation. I didn't want to be too strict with the timeout here, as sometimes it takes ~10 mins to just provision the resources. 
There is an option to reduce it to 20-30 minutes - https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/templates/timeouts.libsonnet#L23 . We could explore those in the next set of changes too.